spec/integ/matrix-to-irc.spec.js

```diff
@@ -534,6 +534,53 @@ describe("Matrix-to-IRC message bridging", function() {
         await p;
     });
 
+    it("should bridge multiline matrix replies without losing information (GH-1198)", async function() {
+        // Trigger an original event
+        await env.mockAppService._trigger("type:m.room.message", {
+            content: {
+                body: "This is the real message",
+                msgtype: "m.text"
+            },
+            room_id: roomMapping.roomId,
+            sender: repliesUser.id,
+            event_id: "$original:bar.com",
+            origin_server_ts: 1_000,
+            type: "m.room.message"
+        });
+        const p = env.ircMock._whenClient(roomMapping.server, testUser.nick, "say",
+            (client, channel, text) => {
+                expect(client.nick).toEqual(testUser.nick);
+                expect(client.addr).toEqual(roomMapping.server);
+                expect(channel).toEqual(roomMapping.channel);
+                expect(text).toContain('Line one');
+                expect(text).toContain('Line one');
```

Comment:
Oh, except this.

Reply:
Ugh. Another example of why the first step of TDD is what it is :P

Thankfully it was working anyway. Amended and forcepushed, should be good now.